src/services/user.services.js

```diff
@@ -41,7 +41,11 @@ const login = async (users) => {
 const register = async (users) => {
   const requestParams = generateParams(users);
   const result = await fetch(`${BACKEND_URL}${routes.BackEndSignUp}`, requestParams);
-  return getResponse(result);
+  const user = await getResponse(result);
+  if (user.error) {
```

Comment:
What is the use of throwing the error here?

Reply:
It will be caught in the action files to track the success and failure of the attempt made either to login or to register